MediaBrowser.Controller/Net/BasePeriodicWebSocketListener.cs

```diff
@@ -98,19 +90,10 @@ private void Start(WebSocketMessageInfo message)
             var dueTimeMs = long.Parse(vals[0], UsCulture);
             var periodMs = long.Parse(vals[1], UsCulture);
 
-            if (vals.Length > 2)
-            {
-                ParseMessageParams(vals.Skip(2).ToArray());
-            }
-
             var cancellationTokenSource = new CancellationTokenSource();
 
             Logger.LogDebug("{1} Begin transmitting over websocket to {0}", message.Connection.RemoteEndPoint, GetType().Name);
 
-            var timer = SendOnTimer ?
```

Comment:
what this timer did and why do you drop it?

Reply:
It was unused and I think it's weird to expose websockets to plugins, so I saw nothing wrong with removing it.